src/api/clothingItems.ts

```diff
@@ -31,11 +34,24 @@ const configureClothingItemEndpoints = (app: Express) => {
                     size: size,
                     item_image_url: image,
                     set_id: set_id,
-                }
-                await createItem(attributes)
-                res.send('Item posted successfully.')
+                };
+                await createItem(attributes);
```

Comment:
Saw you adjusted the createItem method -- are you able to return the created object?

Reply:
updated in new commit